docs/pages/access-controls/access-graph/gitlab.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Discover GitLab Access Patterns with Teleport Policy
+description: Describes how to syncronize GitLab access patterns using Teleport Policy and Access Graph.
+---
+Teleport Policy streamlines and centralizes access management across your entire infrastructure, allowing you to view
+access relationships within seconds. You can see unified, up-to-date relationships and policies between all users, groups,
+and computing resources.
```

Comment:
Nitpick but I would remove this paragraph. The less "happy talk" the better. The next one reads like a very good intro already.
```suggestion
```